README.md

```diff
@@ -365,3 +364,14 @@ to retrieve the OCSP status of a certificate, run Pebble with a `pebble-config.j
 ```
   "ocspResponderURL": "http://127.0.0.1:4002",
 ```
+
+### Listing orders
+
+[As opposed to Boulder](https://github.com/letsencrypt/boulder/issues/3335), Pebble has support
+for enumerating all orders for an ACME account object according to
+[RFC 8555, Section 7.1.2](https://tools.ietf.org/html/rfc8555#section-7.1.2.1). By default, three
+orders are returned per page, to make it easy to test pagination. This number can be modified by
+setting the `PEBBLE_WFE_ORDERS_PER_PAGE` environment variable to a positive integer. For example,
```

Comment:
Do you think there's an advantage to making this configurable? At first glance it feels unnecessary but I might be overlooking something that makes this desirable from a client developer perspective.

Reply:
I've thought about this a bit; I think it would make sense to make it configurable, but I would probably allow all such options to be configurable (via the management interface). I thought it's probably best to come up with a uniform interface for updating (or also querying) configuration values, but I don't think it should happen in this PR.

Reply:
Sounds good. I don't feel strongly about this so I'm happy to leave it as-is in this PR.